### PR TITLE
docs: integrate pouch with dragonfly

### LINKED ISSUE #103: [doc request] how to integrate pouch with dragonfly
**Issue Description**

[Dragonfly](https://github.com/alibaba/dragonfly) is a useful P2P container image distribution software tool which fits for very large size of cluster, maybe 2000+ node. 

We need document to guide us:

* how to integrate pouch with dragonfly;
* then how to use them.
* performance test result if there is.

### PR DESCRIPTION
**Pull request by Ace-Tang**
### Ⅰ. Describe what this PR did

add docs to describe how to run pouch with dragonfly

### Ⅱ. Does this pull request fix one issue?

fixes #103

### CODE REVIEW

File: docs/features/pouch_with_dragonfly.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# Pouch with Dragonfly
+
+Container technology is helpful to facilitate IT operation and maintenance, but the distribution of images has brought a huge challenge at the same time. The images can be as large as serval Gibs, pulling images will be very slow, not to metion the situation lots of pulling requests simultaneously or low network bandwidth. Dragonfly can play a important role here, as a P2P technology, it provides very efficient distribution, avoiding images distribution as a bottleneck in container technology.
```
Comment (allencloud) [inline docs/features/pouch_with_dragonfly.md]: s/serval/serveral
s/metion/mention
s/play a important/play an important/

File: docs/features/pouch_with_dragonfly.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+
+# What is Dragonfly
+
+[Dragonfly](https://github.com/alibaba/Dragonfly#installation) is a P2P file distribution system. It solves the problems of time-consuming distribution, low success rate, and wasteful bandwidth in large-scale file distribution scenarios. Dragonfly significantly improve service capabilities such as data preheating, and large-scale container image distribution.
```
Comment (allencloud) [inline docs/features/pouch_with_dragonfly.md]: s/improve/improves

**Comment by Ace-Tang** [comment]
@allencloud , updated.

**Comment by allencloud** [comment]
LGTM

**Pull request merged**